vulnerabilities/msgvalue-loop.md

```diff
@@ -0,0 +1,44 @@
+# Using ``msg.value`` in a Loop
```

Comment:
Will have to merge/rebase to remove this file

Reply:
Fixed in the latest commit!

Reply:
Looks like it's still included for some reason. Just me?

Reply:
Hi, nothing seems to work even after merging and rebasing. I think it is because one commit from msgValue got included  here and it is causing the problem! As this PR has no conflicts with the base branch, they can be merged and msgvalue-loop.md content hasn't been changed in this PR.